src/components/policy/policy_external/src/cache_manager.cc

```diff
@@ -1389,6 +1390,16 @@ const policy::VehicleInfo CacheManager::GetVehicleInfo() const {
   return vehicle_info;
 }
 
+const utils::OptionalVal<bool> CacheManager::LockScreenDismissalEnabledState() const {
+  CACHE_MANAGER_CHECK(utils::OptionalVal<bool>(utils::OptionalVal<bool>::EMPTY));
+  sync_primitives::AutoLock auto_lock(cache_lock_);
+  policy_table::ModuleConfig& module_config = pt_->policy_table.module_config;
+  if (module_config.lock_screen_dismissal_enabled.is_initialized()) {
+    return utils::OptionalVal<bool>(*module_config.lock_screen_dismissal_enabled);
+  }
+  return utils::OptionalVal<bool>(utils::OptionalVal<bool>::EMPTY);
```

Comment:
@LuxoftAKutsan would be better to create initial empty variable and return it in CACHE_MANAGER_CHECK and outside the if. If it is possible to use this variable inside of, wold be wonderful

Reply:
@LuxoftAKutsan fixed 937d867e865f6bfc3894d297f9c37a5f0f685a11